build/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.5
+
+MAINTAINER Peter Mattis <peter@cockroachlabs.com>
+
+RUN curl --silent --location https://deb.nodesource.com/setup_4.x | bash - && \
+ apt-get dist-upgrade -y && \
+ apt-get install --no-install-recommends --auto-remove -y git build-essential file nodejs && \
+ apt-get clean autoclean && \
+ apt-get autoremove -y && \
+ rm -rf /tmp/*
+
+COPY bower.json npm-shrinkwrap.json package.json tsd.json Makefile ./
+
+RUN make bower.installed npm.installed tsd.installed && \
+ rm bower.json npm-shrinkwrap.json package.json tsd.json Makefile && \
```

Comment:
Perhaps we should leave `bower.json` et-al around and verify in `circle-deps.sh` that they are unchanged in the repo and output an informative error that a new `cockroachdb/builder` image needs to be generated.